setup.py

```diff
@@ -41,6 +41,7 @@
         'termcolor == 1.*',
         'natsort == 5.0.*',
         'appdirs == 1.4.*',
+        'python-dateutil'
```

Comment:
Can you put the current `major.minor` version in here, please? So we don't install new major updates and break things.